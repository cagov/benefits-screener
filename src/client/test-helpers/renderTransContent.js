import React from "react";
import { mount } from "enzyme";
import { I18nextProvider } from "react-i18next";
import i18n from "./i18nForTests";

function renderTransContent(Component, props = {}) {
  const wrapper = mount(
    <I18nextProvider i18n={i18n}>
      <Component {...props} />
    </I18nextProvider>
  );

  // Filter out the very verbose I18nextProvider component wrapper
  return wrapper.find(Component);
}

module.exports = renderTransContent;
