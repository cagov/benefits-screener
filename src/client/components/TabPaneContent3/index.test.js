import renderTransContent from "../../test-helpers/renderTransContent";
import TabPaneContent from "./index";

describe("<TabPaneContent3 />", () => {
  it("renders the TabPaneContent component", async () => {
    const content = renderTransContent(TabPaneContent);
    expect(content).toMatchSnapshot();
  });
});
