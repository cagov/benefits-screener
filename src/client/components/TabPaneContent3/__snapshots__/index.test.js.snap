// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabPaneContent3 /> renders content pane 1`] = `
<div>
  <h3>
    1. We'll confirm your application and weekly benefit amount (WBA) via email and postal mail
  </h3>
  <ul>
    <li>
      <strong>
        You should receive the confirmation of your application within two weeks via postal mail.
      </strong>
       
      We'll let you know what your WBA is, based on our wage records. For an initial estimate of your WBA, use our
       
      <a
        href="https://www.edd.ca.gov/Unemployment/UI-Calculator.htm"
      >
        UI benefit calculator
      </a>
      .
    </li>
    <li>
      You may receive an email confirming that you were automatically registered in UI Online. This allows you to fully access information about your claim and to certify for continued benefits online.
    </li>
    <li>
      If you're not automatically registered in UI Online, you’ll receive a notice in the mail with your EDD Customer Account Number. Use the account number to finish registering for full UI Online access.
    </li>
  </ul>
  <p>
    If we need more information from you to complete your claim, we may call you or mail you a notice. If we are unable to verify your identity or your wages, our system might determine your estimated WBA to be $0.00. If this happens, we may need more time to process your claim. If you were paid wages during the four quarters shown on your
     
    <i>
      Notice of Unemployment Insurance Award
    </i>
    , you should contact the EDD to request a wage investigation.
  </p>
  <h4>
    For business owners, independent contractors, self-employed workers, freelancers, gig workers:
  </h4>
  <p>
    <strong>
      If you qualify for PUA, your initial weekly benefit amount will be $167
    </strong>
     
    for claims starting February 2, 2020. Claims between March 29 to July 25, 2020 will have an additional $600. Your total benefits will last for 39 weeks (including any regular UI and extended benefits you might qualify for).
  </p>
  <p>
    <strong>
      Once we verify your income, your weekly benefit amount may increase
    </strong>
    . Any increase will also apply retroactively. We will notify you of any increase you are entitled to receive.
     
    <a
      href="https://edd.ca.gov/about_edd/coronavirus-2019/pandemic-unemployment-assistance.htm"
    >
      Read more about PUA.
    </a>
  </p>
  <div
    className="highlight-blockquote mb-5"
  >
    <strong>
      Note:
    </strong>
     You will be paid benefits from the date you became unemployed as a result of COVID-19, regardless of when you submitted your claim application.
  </div>
  <h3>
    2. Certify for your first benefit payment
  </h3>
  <div
    className="highlight-blockquote"
  >
    <strong>
      Note:
    </strong>
     Certification is 
    <strong>
      not required
    </strong>
     at this time for benefits with week ending dates of March 14 through May 9, 2020. We'll contact you at a later date to certify for these weeks.
  </div>
  <p>
    When you are required to certify, here’s what you’ll need to do:
  </p>
  <ul>
    <li>
      <strong>
        Certify your claim in UI Online
      </strong>
       once you're registered. You will also be required to certify your claim every two weeks.
    </li>
    <li>
      <strong>
        You can answer "No" when we ask whether you are looking for work, due to COVID-19.
      </strong>
       
      If you meet all other eligibility requirements, you will still receive benefits.
    </li>
  </ul>
  <div
    className="gray-box"
  >
    <h4>
      What is certification?
    </h4>
    <p>
      Certification is the required process of updating the EDD every two weeks with your unemployment status with basic eligibility information:
    </p>
    <ul>
      <li>
        Are you physically able to work?
      </li>
      <li>
        Are you available for work?
      </li>
      <li>
        Are you ready and willing to accept work immediately?
      </li>
      <li>
        Did you refuse work?
      </li>
      <li>
        Did you work and earn wages? If so, how much?
      </li>
    </ul>
    <p>
      The fastest way to certify is on
       
      <a
        href="https://portal.edd.ca.gov/WebApp/Login?resource_url=https%3A%2F%2Fportal.edd.ca.gov%2FWebApp%2FHome"
      >
        UI Online
      </a>
      . You can also do this by phone by calling 1-866-333-4606, or by mailing the paper form.
    </p>
    <div
      className="highlight-blockquote"
    >
      <strong>
        Note:
      </strong>
       Certification is 
      <strong>
        not required
      </strong>
       
      at this time for benefits with week ending dates of March 14 through May 9, 2020. We'll contact you at a later date to certify for these weeks.
    </div>
  </div>
  <h3>
    3. Upload your resume in CalJOBS℠ (Optional)
  </h3>
  <p>
    We understand that it’s a difficult time to look for a job right now. You are 
    <strong>
      currently not
    </strong>
     required to upload your resume to
     
    <a
      href="https://www.caljobs.ca.gov/vosnet/Default.aspx"
    >
      CalJOBS
    </a>
    , during the COVID-19 pandemic.
  </p>
</div>
`;
